CMSgov/cmcs-eregulations
# EREGCSC-1477 -- Title select on Resources page

Resolves [EREGCSC-1477](https://jiraent.cms.gov/browse/EREGCSC-1477)

**Description**

If multiple titles are available (currently fetched from the `/v3/titles` endpoint), then there will be a Title filter on the Resources page that allows users to switch between titles.

**This pull request changes:**

UPDATED BEHAVIOR:
- Behavior is unchanged if only one title is available
- If there is more than one title available to the site (e.g. 42 and 45), then a new TITLE filter will appear on the Resources page
- If there is more than one Title available, no title will be selected by default
   - User can do keyword/text search and Category filtering without selecting a Title
   - User cannot filter by part/subpart/section before selecting a title
- Title filter is unique and not part of the "other" filters (Part, Subpart, etc)
   - Title does not have a Chip
   - When clicking the "Clear All" Chip, Title is not removed or reset
   - Once you select a Title, you cannot "unselect" it without leaving and returning to the page
- Download CSV results will pull from all resources if no title is selected, but if a title is selected, the results will be limited to that title.

**Steps to manually verify this change**

1. Compare to Figma [here](https://www.figma.com/file/Gx6pz0IxWqGsweFVdezKNe/eRegs-Homepage---Multi-View-Update?node-id=3926-13274&t=vXFRokdXIlh5iLV6-0)
2. Poke around and make sure things work as expected
3. All end to end tests pass
